docs/README-cmake.md

```diff
@@ -315,6 +321,12 @@ if(WIN32)
     )
 endif()
 ```
+On Unix systems, `$<TARGET_FILE:...>` will refer to the dynamic library (or framework),
+and you might need to use `$<TARGET_SONAME_FILE:tgt>` instead.
+
+Most often, you can avoid copying libraries by configuring your project with absolute [`https://cmake.org/cmake/help/latest/variable/CMAKE_LIBRARY_OUTPUT_DIRECTORY.html`]
```

Comment:
Shouldn't this be:
...by configuring your project with [`CMAKE_LIBRARY_OUTPUT_DIRECTORY.html`](https://cmake.org/cmake/help/latest/variable/CMAKE_LIBRARY_OUTPUT_DIRECTORY.html`) and ...

Reply:
```suggestion
Most often, you can avoid copying libraries by configuring your project with absolute [`CMAKE_LIBRARY_OUTPUT_DIRECTORY`](https://cmake.org/cmake/help/latest/variable/CMAKE_LIBRARY_OUTPUT_DIRECTORY.html)
```

Reply:
I want absolute in there. Otherwise CMake will make it relative to the current build directory.